tests/enums/test_color.py

```diff
@@ -50,6 +51,12 @@ def test_from_video_rgb(self) -> None:
         result = Matrix.from_video(clip)
         self.assertEqual(result, Matrix.RGB)
 
+    def test_from_video_property(self) -> None:
+        clip = vs.core.std.BlankClip(format=vs.YUV420P8, width=3840, height=2160)
+        clip = vs.core.std.SetFrameProp(clip, "_Matrix", Matrix.BT709)
```

Comment:
Do you think we should mmake two different tests? One with SetFrameProp and the other with Matrix.BT709.apply 🤔